tests/e2e/TestConstants.ts

```diff
@@ -17,12 +17,26 @@ function getBaseUrl(): string {
     return baseUrl.replace(/\/$/, '');
 }
 
+function getKeycloackUrl(): string {
+    const keycloakUrl: string | undefined = process.env.TS_SELENIUM_KEYCLOAK_URL;
+    if (!keycloakUrl) {
+        return 'http://keycloak-url';
+    }
+
+    return keycloakUrl.replace(/\/$/, '');
+}
+
 export const TestConstants = {
     /**
      * Base URL of the application which should be checked
      */
     TS_SELENIUM_BASE_URL: getBaseUrl(),
 
+    /**
+     * The keycloak URL of the application which should be checked
+     */
+    TS_SELENIUM_KEYCLOAK_URL: getKeycloackUrl(),
```

Comment:
I'm a little bit confused - so this is not the Keycloak URL of Che, but the Keycloak URL of the running application?

Reply:
Fixed